tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -71,9 +77,11 @@ export class Ide {
             await this.driverHelper.waitVisibility(exitCodeNotificationLocator, timeout);
         } catch (err) {
             if (err instanceof error.TimeoutError) {
-                Logger.error(`Ide.waitTaskExitCodeNotificationBoolean wait for notification timed out.`);
+                Logger.error(`Ide.waitTaskExitCodeNotificationBoolean wait for notification timed out after ${timeout}.`);
```

Comment:
Why is the exception not being thrown here?

Reply:
:+1: